nekoyume/Assets/_Scripts/UI/Widget/Popup/AvatarInfoPopup.cs

```diff
@@ -594,14 +595,16 @@ private void ShowTooltip(InventoryItemView view)
                 return;
             }
 
-            var (submitEnabledFunc, submitText, onSubmit) = GetToolTipParams(view.Model);
+            var (submitEnabledFunc, submitText, onSubmit, onSubmitDisable) = GetToolTipParams(view.Model);
```

Comment:
```suggestion
            var (submitEnabledFunc, submitText, onSubmit, onClickBlocked) = GetToolTipParams(view.Model);
```
In my humble opinion, It's better to unify name with `onClickBlocked` of line 625.